DigitalLearningSolutions.Data.Tests/Services/DelegateUploadFileServiceTests.cs

```diff
@@ -0,0 +1,859 @@
+namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using ClosedXML.Excel;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.DataServices.UserDataService;
+    using DigitalLearningSolutions.Data.Exceptions;
+    using DigitalLearningSolutions.Data.Models.DelegateUpload;
+    using DigitalLearningSolutions.Data.Models.Register;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using FakeItEasy;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using Microsoft.AspNetCore.Http;
+    using NUnit.Framework;
+
+    // Note that all tests in this file test the internal methods of DelegateUploadFileService

```

Comment:
Are all tests for the internal methods in this file?

Reply:
Yes, there are no tests for the public method that just calls the two internal methods